### ISSUE TITLE
A process blockage

### ISSUE THREAD

**Issue by Darkluna999** [huginn/huginn#3300]
The Same as #3407
Every once in a while this phenomenon can cause a flow blockage
not only `run agent schedule 'every_1m'`
but also `run agent schedule 'every_2m'` `run agent schedule 'every_5m'`
If it is repeated the same time, it will create a process blockage
This is not caused by any of my processes, even the sample process that has just been deployed will occur after a period of time and will not disappear if it is not reset
Even if I delete the entry directly from the database, it can't be corrected, only by restarting the container

**Comment by Darkluna999** [comment]
Some additions, even if I use a version 2 years ago, this problem still occurs.
But I found a solution, this problem will not occur if I don't connect to an external postgres database?

**Comment by przemo** [comment]
I happen to have the same problem. No extremalnie postgres DB.

**Comment by RafaelSchridi** [comment]
having the same problem, no external DB.
![image](https://github.com/huginn/huginn/assets/11460682/469a63cc-3d47-4468-bed2-cc73cebbda01)

**Comment by ww2283** [comment]
same problem here, latest docker image:
<img width="1194" alt="image" src="https://github.com/huginn/huginn/assets/21956104/cf2017aa-45e5-4319-bd6c-f79699803284">

**Comment by dongore** [comment]
Same here as well.

**Comment by kix2902** [comment]
I've got this problem right now, restarting the container fixed it and all the processes finished in a hurry

**Comment by yalidabuda** [comment]
Yes, restarting the container can fix it, but next time I don't know when it will appear again.

**Comment by virtadpt** [comment]
> Some additions, even if I use a version 2 years ago, this problem still occurs. But I found a solution, this problem will not occur if I don't connect to an external postgres database?

Could it be that Huginn is losing contact with the external database server, and failing to reconnect?  Is there anything along those lines in the production.log file?

**Comment by dongore** [comment]
I'm not using am external database, but in my experience it does happen when the internet connection drops out. So it seems it starts hanging when the connection to an external source fails.

**Comment by Darkluna999** [comment]
So after all this time, do you guys have any solutions to fix this problem?

**Comment by virtadpt** [comment]
I think the only solution any of us have really found was to prevent the database connections from dropping, because that seems to cause it.  That's easier said than done due to how TCP/IP works.
